examples/sql_bank/main.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Tamir Duberstein
+
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/lib/pq"
+
+	"github.com/cockroachdb/cockroach/util/log"
+)
+
+const (
+	maxTransfer = 999
+	numAccounts = 3
+	concurrency = 2
+	aggregate   = true
+)
+
+var (
+	dbName      = "bank"
+	tableName   = pq.QuoteIdentifier("accounts")
```

Comment:
It hurts portability, since you've now hard-coded the postgres quoting function. And more importantly, by using Sprintf to construct queries at all, you increase the risk of sql injection. I think it's more important to avoid the use of Sprintf in queries than to be able to rename tables (in practice table and column names tend to be constant and only the database name varies by deployment), so we should just hard-code all table and column names in our queries.